### PR TITLE
DOC: Add example for Beta documentation

### PR DESCRIPTION
**Pull request by a-vzn**

### CODE REVIEW

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -338,6 +338,16 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        --------
+        >>> np.random.beta(0.5, 0.5, 3) // Scalar values for \\alpha and \\beta, size = 3, results in list with 3 samples from beta distribution
```
Comment (rkern) [inline numpy/random/_generator.pyx:343]: We don't use C++ `//` style comments. Put the text above the code snippet. Please see the other examples for how to format the mix of text and code.

Comment (charris) [reply numpy/random/_generator.pyx:343]: The text also needs to be explicitly wrapped to <= 75 characters.

**Comment by mattip** [comment]
ping @aviralsomani are you intending on moving this forward?

**Comment by rossbar** [comment]
This one's been stale for a bit so I'm going to go ahead and close it. If you want to pick this up again @aviralsomani please feel free to reopen after addressing the comments.

**Pull request closed without merging**